src/main/java/Duke.java

```diff
@@ -1,10 +1,220 @@
+import java.io.InputStreamReader;
+import java.io.BufferedReader;
+import java.io.StringReader;
+import java.util.ArrayList;
+import java.util.Arrays;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+    enum commandHints {
+        LIST,
+        TODO,
+        DEADLINE,
+        EVENT,
+        MARK,
+        EXIT
+    }
+    public static void main(String[] args) throws Exception{
+
+        String NAME = "Luna"; // TENTATIVE
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        bot_functions.shifted_print(bot_functions.greetingString(NAME));
+        boolean exitFlag = false;
+        commandHints currentHint = commandHints.EXIT;
+        ArrayList<list_Entry> user_list = new ArrayList<>();
+
+        while (!exitFlag) {
+            String input_command = br.readLine();
+            if (input_command.equalsIgnoreCase("help")) {
+                switch(currentHint) {
```

Comment:
Hi! Perhaps could configure IDE to align "switch" and "case" on the same indentation, according to the code standard?